docs/csharp/advanced-topics/interop/how-to-access-office-interop-objects.md

```diff
@@ -21,6 +21,9 @@ To complete this walkthrough, you must have Microsoft Office Excel 2007 and Micr
 
 [!INCLUDE[note_settings_general](~/includes/note-settings-general-md.md)]
 
+> [!IMPORTANT]
```

Comment:
I'd much rather see an [include file](https://review.learn.microsoft.com/en-us/help/platform/includes-best-practices?branch=main) that is referenced in these four locations. This would ensure that future changes to one update all four articles.

Reply:
Thanks, that's a good catch.

Updated.